books/urls.py

```diff
@@ -28,4 +29,5 @@
     path('admin/', admin.site.urls),
     path('api/', include(router.urls)),
     path('graphql/', GraphQLView.as_view(graphiql=True)),
+    path('api-token-auth/',views.obtain_auth_token),
 ]
```

Comment:
Have you tested this yet? I think the re_path part is going to be important when setting up any of the rest framework authentications.